app/lib/searchValidator.js

```diff
@@ -4,7 +4,7 @@ function validateSearch(search) {
   let errorMessage = null;
   let searchToReturn = search;
 
-  if (!search) {
+  if ((!search) || ((search) && (search.trim() === ''))) {
```

Comment:
I've gone with `isEmpty(search)` as that is what I'm trying to asses, will push up the code in a bit.